package.json

```diff
@@ -71,6 +71,7 @@
         "@emotion/react": "^11.10.4",
         "@emotion/styled": "^11.10.4",
         "@gouvfr/dsfr": "1.12.1",
+        "@gouvfr/dsfr-chart": "^1.0.0",
```

Comment:
Needs to go in dependencies, not devDependencies or it wont work.

Reply:
I was thinking of that but all the other libraries are in "devDependencies" (not sure why). Maybe it is only use during the build ?

Reply:
Yes, all dev dependencies are used only at build time and `@gouvfr/dsfr` is vendored because we need to apply some marginal transformation to it.